### PR TITLE
VR-5487 get deployed model

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: client/verta/tests/test_endpoint/test_endpoint.py [hunk 1]
```diff
@@ -111,3 +113,20 @@ def test_get_access_token(self, client):
         token = endpoint.get_access_token()
 
         assert token is None
+
+    def test_get_deployed_model(self, client, experiment_run, model_for_deployment):
+        model = model_for_deployment['model'].fit(
+            model_for_deployment['train_features'],
+            model_for_deployment['train_targets'],
+        )
+        experiment_run.log_model_for_deployment(model)
```
Comment (convoliution) [inline client/verta/tests/test_endpoint/test_endpoint.py]: `log_model_for_deployment()` is deprecated.
```suggestion
        experiment_run.log_model(model, custom_modules=[])
        experiment_run.log_requirements(['scikit-learn'])
```

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/verta/verta/_deployment/endpoint.py [hunk 2]
```diff
@@ -187,3 +188,14 @@ def get_access_token(self):
...
+    def get_deployed_model(self):
+        status = self.get_status()
+        if status['status'] != "active":
+            raise RuntimeError("model is not currently deployed (status: {})".format(status))
+
+        access_token = self.get_access_token()
+        return DeployedModel.from_url("{}://{}/api/v1/predict{}".format(
+            self._conn.scheme,
+            self._conn.socket,
+            self.path), access_token)
```
Comment (convoliution) [inline client/verta/verta/_deployment/endpoint.py]: Minor suggestion:
Could we do a `url = "..."` in a separate line? Like [this](https://github.com/VertaAI/modeldb/blob/eafe27d/client/verta/verta/_deployment/endpoint.py#L129-L136).
For me, it's a bit easier to read.

File: client/verta/tests/test_endpoint/test_endpoint.py [hunk 3]
```diff
@@ -111,3 +113,20 @@ def test_get_access_token(self, client):
         token = endpoint.get_access_token()
 
         assert token is None
+
+    def test_get_deployed_model(self, client, experiment_run, model_for_deployment):
```
Comment (convoliution) [inline client/verta/tests/test_endpoint/test_endpoint.py]: Let's use the `created_endpoints` fixture to cleanup after the test ([example](https://github.com/VertaAI/modeldb/blob/412369f/client/verta/tests/test_endpoint/test_endpoint.py#L32-L36)), until I figure out an easier way to handle cleanups...

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/verta/verta/_deployment/endpoint.py (hunk 2 above)
Comment (evgeniilezhnin) [reply client/verta/verta/_deployment/endpoint.py]: extracted url

File: client/verta/tests/test_endpoint/test_endpoint.py (hunk 3 above)
Comment (evgeniilezhnin) [reply client/verta/tests/test_endpoint/test_endpoint.py]: fixed

**Review by convoliution — APPROVED** [review]

**Pull request merged**